cryptol-remote-api/python/tests/cryptol/test_smt.py

```diff
@@ -47,6 +48,38 @@ def test_SMT(self):
         self.assertTrue(ex_partial_sat)
         self.assertIsInstance(ex_partial_sat, cryptol.Satisfiable)
 
+    def test_each_online_solver(self):
+        # we test each solver that is packaged for use with what4
+        # via https://github.com/GaloisInc/what4-solvers
+        ex_true  = '\(x : [8]) -> x+x == x+x'
+        solvers = \
+            [solver.CVC4,
+             solver.YICES,
+             solver.Z3,
+             #solver.BOOLECTOR,
```

Comment:
Are there issues with using these commented-out solvers, out of curiosity?

Reply:
As sneakily noted above these lines, I was trying to test only the solvers that are distributed in `what4-solvers`. The exception at this point being the SBV ones, but I've added those back into the list with a subsequent commit.

Reply:
I added a little additional clarification to the comment regarding the intentionally commented out solvers.